package.json

```diff
@@ -15,7 +16,8 @@
     "test:bundle": "jest --env=jsdom e2e-tests/bundle/**",
     "test:e2e": "jest -c e2e-tests/jest.config.js",
     "test:e2e:start": "serve",
-    "validate": "npm run build && npm run lint && npm test -- --coverage"
+    "typecheck": "tsc",
```

Comment:
Can update this to include `noEmit`:
`"typecheck": "tsc --noEmit"`

Since its just a quality check, no need to emit an output.

See https://www.typescriptlang.org/docs/handbook/compiler-options.html